tool-populate/index.js

```diff
@@ -43,26 +44,25 @@ process.on('exit', exitHandler.bind(null, {log: true}))
 process.on('uncaughtException', exitHandler.bind(null, {exit: true}))
 
 function exitHandler(options, err) {
-    if (err)
-      console.error(err.stack)
-
-    if (program.reset) {
-      if (runningLocal)
-        console.log('Server Reset.')
-      else
-        console.warn('Denying to reset the remote server (' + Parse.serverURL + ')')
-    } else if (program.create) {
-      console.log('Parse server populated.')
-    } else if (program.print) {
-      console.log('Stored Data: ' +
-        Store.surveys.length + ' surveys, ' +
-        Store.forms.length + ' forms, ' +
-        Store.questions.length + ' questions, ' +
-        Store.triggers.length + ' triggers.'
-      )
-    }
+  if (err)
+    console.error(err.stack)
```

Comment:
If there is an error shouldn't it return?  I've seen cases where the output was as follows:

```
Creating Parse server demo data...
Failed to create Survey, with error code: Permission denied for this action.
Parse server populated.
```